lib/views/pages/organization/join_organization.dart

```diff
@@ -188,210 +196,261 @@ class _JoinOrganizationState extends State<JoinOrganization> {
                         filled: true,
                         border: OutlineInputBorder(
                           borderRadius: BorderRadius.circular(15.0),
-                          borderSide:
-                              BorderSide(color: Colors.white, width: 0.0),
+                          borderSide: BorderSide(
+                            color: Colors.white,
+                            width: 0.0,
+                          ),
                         ),
                         enabledBorder: OutlineInputBorder(
                           borderRadius: BorderRadius.circular(15.0),
-                          borderSide:
-                              BorderSide(color: Colors.white, width: 0.0),
+                          borderSide: BorderSide(
+                            color: Colors.white,
+                            width: 0.0,
+                          ),
                         ),
                         prefixIcon: Padding(
                           padding: EdgeInsets.all(0.0),
-                          child: Icon(Icons.search, color: Colors.black),
+                          child: Icon(
+                            Icons.search,
+                            color: Colors.black,
+                          ),
                         ),
                         hintText: "Search Organization Name"),
                   ),
                   SizedBox(height: 15),
                   Expanded(
-                      child: Container(
```

Comment:
@TusharGupta05, How is this change required to fix this issue ?
Please add the changes which fixes the issue.

Reply:
@Sagar2366 sir I have removed all such unnecessary changes.